arangod/Pregel/CommonFormats.h

```diff
@@ -64,6 +64,15 @@ struct HITSValue {
   double hubScore;
 };
 
+/// Value for Hyperlink-Induced Topic Search (HITS; also known as
+/// hubs and authorities) according to the paper
```

Comment:
Which paper?

Reply:
Added a proper reference (here and in `HITSKleinberg.h`).